qiling/debugger/qdb/utils.py

```diff
@@ -121,6 +131,17 @@ def get_cpsr(bits: int) -> (bool, bool, bool, bool):
             )
 
 
+def get_x86_eflags(bits: int) -> (bool, bool, bool, bool, bool, bool):
```

Comment:
This is not valid annotation. You may want to use `Dict[str, bool]` instead.

Reply:
thx for the hint, fixed in following commit.